crt_portal/cts_forms/templates/base.html

```diff
@@ -61,8 +62,12 @@
     <script src="{% static 'js/focus_alert.js' %}"></script>
     <script src="{% static 'js/clear_error_class.js' %}"></script>
     {% block page_js %}{% endblock %}
+
     {% block analytics %}
-    <script async="async" id="_fed_an_ua_tag" src="https://dap.digitalgov.gov/Universal-Federated-Analytics-Min.js?agency=DOJ&amp;sp=find&amp;yt=true"></script>
+    {% environment as env %}
+    {% if env == PRODUCTION %}
+      <script async="async" id="_fed_an_ua_tag" src="https://dap.digitalgov.gov/Universal-Federated-Analytics-Min.js?agency=DOJ&amp;sp=find&amp;yt=true"></script>
```

Comment:
Curious why we have `yt` (youtube) on? Also no subagency?

Reply:
Also not sure about the `sp` query parameter since it's mainly used to track searches. I don't think adding this will break anything but it won't be useful, afaics.

Reply:
CF https://github.com/digital-analytics-program/gov-wide-code/blob/master/documentation/GSA%20DAP%204.1%20-%20Quick%20Guide.pdf -- see `Query String Parameters and Default Values Reference`

Reply:
Those are both great suggestions!